charts/argo-workflows/values.yaml

```
@@ -12,6 +12,14 @@ init:
 
 createAggregateRoles: true
 
+## String to partially override "argo-workflows.fullname" template
+##
+# nameOverride:
+
+## String to fully override "argo-workflows.fullname" template
+##
+# fullnameOverride:
```

I think, we'd better use this without being a comment, so tools like ArgoCD detect this value in the UI :-) Sorry for that, as it was my suggestion to use it like this.
```suggestion
fullnameOverride:
```